package.json

```
@@ -31,27 +30,36 @@
     "release:version": "changeset version && pnpm install --lockfile-only",
     "install:foundry": "curl -L https://foundry.paradigm.xyz | bash && pnpm update:foundry",
     "update:foundry": "bash ./ops/scripts/install-foundry.sh",
-    "install:abigen": "go install github.com/ethereum/go-ethereum/cmd/abigen@$(cat .abigenrc)",
+    "check:foundry": "bash ./packages/contracts-bedrock/scripts/checks/check-foundry-install.sh",
+    "install:kontrol": "curl -L https://kframework.org/install | bash && pnpm update:kontrol",
+    "update:kontrol": "kup install kontrol --version v$(jq -r .kontrol < versions.json)",
```

And please add `check-foundry-install.sh` file.
```suggestion
    "check:foundry": "bash ./packages/contracts/scripts/checks/check-foundry-install.sh",
```